mobile/src/src/utils/endpoints.jsx

```diff
@@ -270,6 +270,10 @@ export class PollsEndpoint extends AbstractEndpoint {
   static endpoint = '/api/polls/';
 }
 
+export class PollOptionsEndpoint extends AbstractEndpoint {
+  static endpoint = '/api/polls/options/';
+}
```

Comment:
This endpoint should only implements list() and post(), not all the functions in AbstractEndpoint. We can prob just copy those 2 functions from AbstractEndpoint and past them in here.